src/components/Global/TokenSelector/NewTokenSelector.tsx

```
@@ -0,0 +1,527 @@
+            filteredByChain = selectedWallet.balances.filter((balance) => {
+                const balanceChainId = String(balance.chainId)
+                const isMatch = balanceChainId === selectedChainID
+
+                return isMatch
+            })
+        }
+
+        const filteredBalances = filteredByChain.filter((balance) => {
+            const hasSymbol = !!balance.symbol
+            const symbolMatch = hasSymbol && balance.symbol.toLowerCase().includes(lowerSearchValue)
+            const nameMatch = balance.name && balance.name.toLowerCase().includes(lowerSearchValue)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)